bindsnet/network/nodes.py

```diff
@@ -181,14 +181,14 @@ def step(self, inpts, dt):
         self.refrac_count[self.refrac_count != 0] -= dt
 
         # Check for spiking neurons.
-        self.s = (self.v >= self.thresh) & (self.refrac_count == 0)
```

Comment:
This `& (self.refrac_count == 0)` is needed. It's not a bug. Neurons shouldn't spike if they're refractory.

Reply:
Actually, I see why this works now. Disregard these comments!